### PR TITLE
Implement CSV downloads for collections

### PR DESCRIPTION
**Pull request by paninee**
https://github.com/participedia/api/issues/1006

### CODE REVIEW

**Comment by jesicarson** [comment]
I'm being nosey, and this is probably in progress, but I don't see the Results CSV button on collections pages yet.

File: public/js/csv-generator.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const csvGenerator = {
```
Comment (ascott) [inline public/js/csv-generator.js:1]: let's fix the white space on this file please.

File: public/js/csv-generator.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+	init() {
```
Comment (ascott) [inline public/js/csv-generator.js:2]: i think this module name could be more clear, what do you think of `downloadCsvButton.js` instead

**Comment by ascott** [comment]
@paninee i just left a couple comments. when you have reviewed/updated, please ship and test on staging and then to prod. thanks!

**Pull request merged**